package.json

```
@@ -0,0 +1,24 @@
+{
+    "devDependencies": {
+        "@symfony/webpack-encore": "^0.31.0",
```

```suggestion
        "@babel/core": "^7.12.3",
        "@symfony/webpack-encore": "^0.31.0",
```
To resolve some dependency warnings after `yarn install`